apps/chat/src/app.module.ts

```diff
@@ -0,0 +1,49 @@
+import { Module } from '@nestjs/common';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { MongooseModule, MongooseModuleOptions } from '@nestjs/mongoose';
+import { TypeOrmModule, TypeOrmModuleOptions } from '@nestjs/typeorm';
+import { ChatModule } from './chat/chat.module';
+import mongoConfig from './config/mongo.config';
+import mysqlConfig from './config/mysql.config';
+
+@Module({
+  imports: [
+    ConfigModule.forRoot({
+      isGlobal: true,
+      load: [mysqlConfig, mongoConfig],
+    }),
+    TypeOrmModule.forRootAsync({
+      inject: [ConfigService],
+      useFactory: async (configService: ConfigService) => {
+        const boardTypeOrmModuleOptions: TypeOrmModuleOptions = {
+          type: 'mysql',
+          host: configService.get('mysql.host'),
+          port: configService.get('mysql.port'),
+          database: configService.get('mysql.database'),
+          username: configService.get('mysql.username'),
+          password: configService.get('mysql.password'),
+          autoLoadEntities: true,
+          entities: [`${__dirname}/**/*.entity.{ts,js}`],
+          logging: true,
+          synchronize: process.env.NODE_ENV === 'development',
+        };
+        return boardTypeOrmModuleOptions;
+      },
+    }),
+    MongooseModule.forRootAsync({
+      inject: [ConfigService],
+      useFactory: async (configService: ConfigService) => {
+        const gatewayMongooseOptions: MongooseModuleOptions = {
+          uri: `${configService.get('mongo.host')}:${configService.get('mongo.port')}`,
+          auth: {
+            username: configService.get('mongo.username'),
+            password: configService.get('mongo.password'),
```

Comment:
```suggestion
        const chatMongooseOptions: MongooseModuleOptions = {
          uri: `${configService.get('mongo.host')}:${configService.get('mongo.port')}`,
          auth: {
            username: configService.get('mongo.username'),
            password: configService.get('mongo.password'),
```
이 부분 도 chat 에 관한 Mongoose 선언을 인식 할 수 있도록
chatMongooseOptions 로 바뀌어도 좋을 거 같습니다

Reply:
넵 chat 서비스에 맞게 적절한 이름으로 변경하겠습니다!